Remove h2 css that overrides the theme css

janette opened the pull request:
## Description

The dkan_dataset module css overrides the font weight, size and color values that should be handled by the theme.

> fonts on search, groups, and groups page are not align with font your face choices
## Acceptance Criteria
- [ ] Titles on search results are determined by the theme (use link color and font-weight=300) and not hard-coded into the dkan_dataset module css;
- [ ] When on a resource page, the hover state of the first and last main menu items are not rounded

Commit:
New release replacements on makefiles

dafeder committed:
Update dkan_dataset.make

janette committed:
Update dkan_dataset_content_types with new absolute_url link settings

janette committed:
Remove view h2 css that overrides the theme css

dafeder commented:
Sounds fine in theory, but don't love how the datasets list looks now. Is there a corresponding PR for the theme?
![2016-06-03_12-04-36](https://cloud.githubusercontent.com/assets/309671/15785052/5b2577d2-2983-11e6-9fad-d62a9d1baa51.png)

janette commented:
This is how it will look with the changes to nuboot_radix

![screenshot_6_3_16__11_16_am](https://cloud.githubusercontent.com/assets/314172/15786133/96b9d534-2980-11e6-8515-38d602c1ca57.png)

janette committed:
Remove css that sets the margin on search results titles

dafeder committed:
Update version numbers

janette committed:
Update css for resource content - fix list-group-item style to exclude main menu

teosibileau committed:
Issue #CIVIC-2719
+ Adds support for dkan archives to dkan-module-init
+ Updates ci recipe for new structure

teosibileau committed:
Expand memory for circle

dafeder committed:
Update version numbers

dafeder committed:
Reset version numbers
